test/.gitignore

```diff
@@ -2,3 +2,5 @@
 *.bc
 *.o
 *.bpl
+corral_out_trace.txt
```

Comment:
Do we need these? The way we are invoking Corral now, I think that no output files should be generated by Corral. Am I wrong about that?